### ISSUE TITLE
Update interval default is too low

### ISSUE THREAD

**Issue by ramonsmits** [Particular/NServiceBus.Metrics.PerformanceCounters#16]
The current update interval is too low resulting in jaggy lines in Perfmon.

After testing the following configuration with a reporting interval of 100 milliseconds the graph looks very similar to performance counters of previous versions in core.

```
endpointConfiguration
    .EnableWindowsPerformanceCounters()
    .UpdateCounterEvery(TimeSpan.FromMilliseconds(100));
```

![image](https://user-images.githubusercontent.com/152998/27220112-3dc6bcfc-5284-11e7-8779-3693237463db.png)

**Comment by Scooletz** [comment]
Will be addressed by #20

**Comment by Scooletz** [comment]
#19 is merged.
